lib/src/Uncertainty/Algorithm/MetaModel/Kriging/GeneralLinearModelAlgorithm.cxx

```diff
@@ -431,8 +436,13 @@ void GeneralLinearModelAlgorithm::setCovarianceModel(const CovarianceModel & cov
   const UnsignedInteger optimizationDimension = reducedCovarianceModel_.getParameter().getSize();
   if (optimizationDimension > 0)
   {
-    const Point lowerBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationLowerBound"));
-    const Point upperBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationUpperBound"));
+    Point lowerBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationLowerBound"));
+    Point upperBound(optimizationDimension, ResourceMap::GetAsScalar( "GeneralLinearModelAlgorithm-DefaultOptimizationUpperBound"));
+    if (normalize_)
+    {
+      lowerBound = inputTransformation_(lowerBound);
+      upperBound = inputTransformation_(upperBound);
+    }
```

Comment:
@dbarbier what about  GeneralLinearModelAlgorithm::setOptimizationBounds ?

Reply:
It depends whether we consider that set/get works on normalized or original bounds.  With this commit, we consider that accessors deal with normalized bounds.  I push another commit to do what you suggest.